PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 | Q               | A
 | --------------- | ---
-| Bug fix?        | no|yes
-| New feature?    | no|yes
-| BC breaks?      | no|yes
+| Bug fix?        | no\|yes
+| New feature?    | no\|yes
+| BC breaks?      | no\|yes
```

Comment:
Sure, but the point was not to change how PR table looks like, but only to fix it :) I have no problem with slash and could change it in a minute ;)
![zrzut ekranu 2016-11-07 o 09 38 12](https://cloud.githubusercontent.com/assets/6212718/20051175/f5ac3bc2-a4cd-11e6-981d-fd293b803c79.png)